### PR TITLE
Layout added for stanrc85

### PR DESCRIPTION
**Pull request by stanrc85**
Initial upload of stanrc85 60% layout

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: layouts/community/60_ansi/stanrc85-ansi/keymap.c [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
+#define _____ KC_TRNS //readability
```
Comment (noroadsleft) [inline layouts/community/60_ansi/stanrc85-ansi/keymap.c]: QMK defines a keycode for this automatically.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L29-L30

Please delete Line 19, and replace all instances of  
`_____` (five underscores) with  
`_______` (seven underscores).

Comment (stanrc85) [reply layouts/community/60_ansi/stanrc85-ansi/keymap.c]: I've made the requested changes.

File: layouts/community/60_ansi/stanrc85-ansi/config.h [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#define TAPPING_TERM 200
```
Comment (drashna) [inline layouts/community/60_ansi/stanrc85-ansi/config.h]: This should/needs to have an include guard.

```suggestion
#pragma once

#define TAPPING_TERM 200
```

File: layouts/community/60_ansi/stanrc85-ansi/rules.mk [hunk 3]
```diff
@@ -0,0 +1,2 @@
+USER_NAME := stanrc85-ansi
```
Comment (drashna) [inline layouts/community/60_ansi/stanrc85-ansi/rules.mk]: Unless you're using a userspace (and it doesn't appear that you are), this is not necessary and should be removed.

```suggestion
```

File: layouts/community/60_ansi/stanrc85-ansi/keymap.c [hunk 4]
```diff
@@ -0,0 +1,70 @@
+/* Copyright 2018 MechMerlin
```
Comment (mechmerlin) [inline layouts/community/60_ansi/stanrc85-ansi/keymap.c]: =)

File: layouts/community/60_ansi/stanrc85-ansi/config.h (hunk 2 above)
Comment (stanrc85) [reply layouts/community/60_ansi/stanrc85-ansi/config.h]: Changes made.

File: layouts/community/60_ansi/stanrc85-ansi/rules.mk (hunk 3 above)
Comment (stanrc85) [reply layouts/community/60_ansi/stanrc85-ansi/rules.mk]: Changes made.

File: layouts/community/60_ansi/stanrc85-ansi/keymap.c (hunk 4 above)
Comment (drashna) [reply layouts/community/60_ansi/stanrc85-ansi/keymap.c]: Shhh :D

Comment (stanrc85) [reply layouts/community/60_ansi/stanrc85-ansi/keymap.c]: Just ignore that part :)

Comment (drashna) [reply layouts/community/60_ansi/stanrc85-ansi/keymap.c]: Ideally, you'd want to add your name, or change the name here. 

If you're fine with leaving this, and @mechmerlin doesn't mind, then we can leave it as is.

File: layouts/community/60_ansi/stanrc85-ansi/readme.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+Stanrc85's Standard ANSI 60% Layout
```
Comment (drashna) [inline layouts/community/60_ansi/stanrc85-ansi/readme.md]: This is going to bug me. 

This should be:
```suggestion
# Stanrc85's Standard ANSI 60% Layout
```

And the following line could be removed

Comment (stanrc85) [reply layouts/community/60_ansi/stanrc85-ansi/readme.md]: Fixed

**Comment by drashna** [comment]
Awesome!  Thanks!

**Pull request merged**